### PR TITLE
doc: Update squelching description in the SPEC doc

### PR DESCRIPTION
**Pull request by Rqnsom**
- Also: Test index symbolic language legend update.

### CODE REVIEW

File: SPEC.md [hunk 1]
```diff
@@ -205,26 +215,26 @@ The test index makes use of symbolic language in describing connection and messa
...
+    -> mtSQUELCH (B)
+    << mtPROPOSE_LEDGER (A)
+
+    Assert: A synthetic node receives only mtPROPOSE_LEDGER messages with a key from node 1
+    after squelching node public keys belonging to nodes 2 and 3 (B).
 
 ### ZG-CONFORMANCE-017
-    
+
     The node sends mtMANIFESTS after the handshake.
```
Comment (niklaslong) [inline SPEC.md:234]:
```suggestion
    The node sends mtMANIFESTS after the handshake.
    
```

There are many test cases where this space is missing, I think it makes the cases more readable.

File: src/tests/conformance/cmd/squelch.rs [hunk 2]
```diff
@@ -15,9 +15,7 @@
 //!     <- mtPROPOSE_LEDGER (validator1)
 //!     <- mtPROPOSE_LEDGER (validator2)
 //!     -> mtSQUELCH (validator1)
-//!     <-
 //!     <- mtPROPOSE_LEDGER (validator2)
-//!     <-
```
Comment (Rqnsom) [inline src/tests/conformance/cmd/squelch.rs:20]: How would you improve this example? Or we just remove this example from here and that's it 😀

File: SPEC.md [hunk 3]
```diff
@@ -147,9 +155,11 @@ The test index makes use of symbolic language in describing connection and messa
 
     The node should ignore the squelch message for its validator public key.
     <>
```
Comment (niklaslong) [inline SPEC.md:157]:
```suggestion
    
    <>
```

File: SPEC.md [hunk 4]
```diff
@@ -73,7 +73,15 @@ The fuzz tests aim to buttress the message conformance tests with extra verifica
 
 # Test Index
 
-The test index makes use of symbolic language in describing connection and message sending directions. As a convention, Ziggurat test nodes are to the left of the connection/message arrows, and Rippled instances are to the right: `A -> B` and `A <- B`. In this way, `->` signifies "Ziggurat connects to Rippled" and `<-` signifies the opposite. Furthermore, `-> ping` signifies "Ziggurat sends a `Ping` message to Rippled" and `<- pong` signifies "Rippled sends a `Pong` message to Ziggurat". Lastly, `<>` signifies a completed handshake, in either direction.
+The test index makes use of symbolic language in describing connection and message sending directions.
+
+| Symbol | Meaning |
```
Comment (niklaslong) [inline SPEC.md:78]: Nice 👍

**Review by niklaslong — APPROVED** [review]
Left a few suggestions, otherwise lgtm 👍

File: SPEC.md (hunk 4 above)
Comment (Rqnsom) [reply SPEC.md:78]: Let's eventually align all three repos with the same table

File: SPEC.md (hunk 3 above)
Comment (Rqnsom) [reply SPEC.md:159]: Done

File: SPEC.md (hunk 1 above)
Comment (Rqnsom) [reply SPEC.md:238]: Done everywhere

File: src/tests/conformance/cmd/squelch.rs (hunk 2 above)
Comment (Rqnsom) [reply src/tests/conformance/cmd/squelch.rs:20]: Done

**Pull request merged**